raiden/tests/integration/network/transport/rif_comms/node.py

```diff
@@ -39,26 +39,29 @@ class Node:
     """
 
     def __init__(self, config: Config):
-        self.address = config.address
+        self.clients = list()
         self._api_endpoint = config.api_endpoint
         self._env_name = config.env_name
-
-        self.client = RIFCommsClient(rsk_address=self.address, grpc_api_endpoint=self._api_endpoint)
+        for i in range(config.amount_of_clients):
+            self.clients.append(RIFCommsClient(rsk_address=generate_address(), grpc_api_endpoint=self._api_endpoint))
         self._process = self.start()
 
     def start(self) -> Popen:
         """
         Start a RIF Comms node process and connect to it.
         """
         process = CommsProcess.start(env_name=self._env_name)
-        self.client.connect()
+        for client in self.clients:
+            client.connect()
         return process
 
     def stop(self):
         """
         Disconnect from RIF Comms node program, and stop its process.
         """
         try:
-            self.client.disconnect()
+            for client in self.clients:
```

Comment:
Maybe an except could be usefull here and maybe adding a log with the exception, or we are trowing that to the top level?

Reply:
i agree that if the disconnect fails on the client therefore test must fail, so i would keep it like that